docs/api/rvdss.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Respiratory Virus Detections in Canada
+parent: Data Sources and Signals
+grand_parent: Other Endpoints (COVID-19 and Other Diseases)
+nav_order: 1
+---
+
+# Respiratory Virus Detections in Canada
+
+{: .no_toc}
+
+- **Source name:** `rvdss`
+- **Earliest issue available:** Epiweek 36 2013 (2013-09-07)
+- **Date of last change:** Never
+- **Available for:** province, lab, region, nation (see [Geography](#geography))
+- **Time type:** week (see [date format docs](../covidcast_times.md))
+- **License:** [Open Government Licence - Canada](#source-and-licensing)
+
+This data source of respiratory virus detections in Canada is collected by the [Respiratory Virus Detection Surveillance System (RVDSS)](https://health-infobase.canada.ca/respiratory-virus-surveillance/about.html) and published by the Public Health Agency of Canada (PHAC). Laboratory tests for various respiratory illnesses are reported on a weekly basis by sentinel laboratories across Canada to the [Centre for Immunization and Respiratory Infectious Diseases (CIRID)](https://www.canada.ca/en/public-health/services/infectious-diseases/centre-immunization-respiratory-infectious-diseases-cirid.html), a subsidiary of PHAC. The data was originally reported in weekly online reports, but since June 2024, has changed to being reported through a dynamic dashboard.
+
+NOTE: Pandemic COVID-19 is not reported in this data source.
+
+| Signal                         | Description                                                                                                                                 |
```

Comment:
suggestion: we should add the descriptive columns (time_value, issue, etc) to this table so that we can define them and explain the general time formats.